client/firewall/iptables/rulestore_linux.go

```diff
@@ -37,6 +37,11 @@ func (s *ipList) UnmarshalJSON(data []byte) error {
 		return err
 	}
 	s.ips = temp.IPs
+
+	if temp.IPs == nil {
```

Comment:
Consider this example: https://go.dev/play/p/h7pdJWXBhL3

A user could manipulate the state file json, change the field to `null` and the map would be overwritten with nil.It will panic if we try to write to it.

Reply:
Good to know!